index.js

```diff
@@ -45,10 +49,10 @@ function configureFormatters(env) {
  * @param {String} env The name of the environment to use. This affects formatting.
  * @param {String} level The level to use when setting the logger up.
  */
-function setupNodeLogger({ env = '', level = DEFAULT_LOG_LEVEL }) {
```

Comment:
It seems to me like compact would be the indication for using the simple logger, but it seems like the other way around.

What am I missing?

Reply:
maybe a different name would be better
so far i thought of compact and verbose. maybe just call it json = true/false and that's it.

Reply:
Yeah. json makes sense, I think.